### ISSUE TITLE
Jest tests are not aware of internal Next.js fork's next/image configured domains

### ISSUE THREAD

**Issue by wescopeland** [blitz-js/blitz#2192]
### What is the problem?

If a component is using `next/image` or the new `Image` import from `blitz`, an error will be thrown on Jest tests stating that the `src` prop is invalid due to the domain not being specified in the _next.config.js_ file. However, this occurs when the domain is indeed specified in _blitz.config.js_ file.

### Paste all your error logs here:

```
 ● Component: UserFeedListItem › renders without crashing

    Invalid src prop (https://images.igdb.com/img.png) on `next/image`, hostname "images.igdb.com" is not configured under images in your `next.config.js`
    See more info: https://nextjs.org/docs/messages/next-image-unconfigured-host

      60 |   }
      61 |
    > 62 |   return defaultRender(ui, { wrapper, ...options });
         |          ^
      63 | }
      64 |
      65 | // --------------------------------------------------

      at load (node_modules/@blitzjs/next/client/image.tsx:578:15)
      at loader (node_modules/@blitzjs/next/client/image.tsx:203:12)
      at map (node_modules/@blitzjs/next/client/image.tsx:174:14)
          at Array.map (<anonymous>)
      at generateImgAttrs (node_modules/@blitzjs/next/client/image.tsx:172:8)
      at Image (node_modules/@blitzjs/next/client/image.tsx:443:17)
      at renderWithHooks (node_modules/react-dom/cjs/react-dom.development.js:15934:18)
      at mountIndeterminateComponent (node_modules/react-dom/cjs/react-dom.development.js:19156:13)
      at beginWork (node_modules/react-dom/cjs/react-dom.development.js:20651:16)
      at beginWork$1 (node_modules/react-dom/cjs/react-dom.development.js:25980:14)
      at performUnitOfWork (node_modules/react-dom/cjs/react-dom.development.js:25179:12)
      at workLoopSync (node_modules/react-dom/cjs/react-dom.development.js:25095:5)
      at renderRootSync (node_modules/react-dom/cjs/react-dom.development.js:25054:7)
      at performSyncWorkOnRoot (node_modules/react-dom/cjs/react-dom.development.js:24652:18)
      at scheduleUpdateOnFiber (node_modules/react-dom/cjs/react-dom.development.js:24237:7)
      at updateContainer (node_modules/react-dom/cjs/react-dom.development.js:27519:14)
      at node_modules/react-dom/cjs/react-dom.development.js:28111:7
      at unbatchedUpdates (node_modules/react-dom/cjs/react-dom.development.js:24789:12)
      at legacyRenderSubtreeIntoContainer (node_modules/react-dom/cjs/react-dom.development.js:28110:5)
      at Object.render (node_modules/react-dom/cjs/react-dom.development.js:28193:10)
      at node_modules/@testing-library/react/dist/pure.js:99:25
      at batchedUpdates$1 (node_modules/react-dom/cjs/react-dom.development.js:24740:12)
      at act (node_modules/react-dom/cjs/react-dom-test-utils.development.js:1038:14)
      at render (node_modules/@testing-library/react/dist/pure.js:95:26)
      at render (test/utils.tsx:62:10)
      at Object.<anonymous> (app/user/components/UserFeedListItem/UserFeedListItem.test.tsx:26:27)
```

### Paste all relevant code snippets here:

```typescript
import { Image } from "blitz";

const TestComponent = () => {
  return (
    <Image
      src="https://images.igdb.com/img.png"
      alt="this is a fake image as an example, not a real url"
      width={122}
      height={68}
      objectFit="cover"
      objectPosition="50% 30%"
    />
  )
}
```

```
// blitz.config.js

const { sessionMiddleware, simpleRolesIsAuthorized } = require("blitz");

const blitzConfig = {
  middleware: [
    sessionMiddleware({
      isAuthorized: simpleRolesIsAuthorized
    })
  ],

  images: {
    domains: [
      "images.igdb.com",
    ]
  },

  experimental: {
    reactMode: "legacy"
  }
};

module.exports = blitzConfig;
```

```typescript
describe("Component: TestComponent", () => {
  it("renders without crashing", () => {
    // ARRANGE
    const { container } = render(
      <TestComponent />
    );

    // ASSERT
    expect(container).toBeTruthy(); // this will fail
  });
});
```

### What are detailed steps to reproduce this?

1. Create a component that uses the `Image` component from Next.js. Point it to an external domain.
2. Write a test in Jest for this component.

### Run `blitz -v` and paste the output here:

```
macOS Big Sur | darwin-x64 | Node: v14.16.0

blitz: 0.34.0-canary.2 (global)
blitz: 0.34.0-canary.2 (local)

  Package manager: yarn 
  System:
    OS: macOS 11.2.3
    CPU: (8) x64 Intel(R) Core(TM) i7-1068NG7 CPU @ 2.30GHz
    Memory: 1018.84 MB / 32.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.16.0 - /usr/local/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 6.14.11 - /usr/local/bin/npm
    Watchman: Not Found
  npmPackages:
    @prisma/client: 2.19.0 => 2.19.0 
    blitz: 0.34.0-canary.2 => 0.34.0-canary.2 
    prisma: 2.19.0 => 2.19.0 
    react: 0.0.0-experimental-6a589ad71 => 0.0.0-experimental-6a589ad71 
    react-dom: 0.0.0-experimental-6a589ad71 => 0.0.0-experimental-6a589ad71 
    typescript: 4.1.5 => 4.1.5 
```

**Comment by flybayer** [comment]
Does this happen even after clearing node_modules and reinstalling?

**Comment by flybayer** [comment]
Also did this work `<= 0.33`?

**Comment by wescopeland** [comment]
This seems to have been fixed in `0.34.0`.
